examples/jupyter/import_config_xarray.ipynb

```diff
@@ -70,19 +71,18 @@
       "    number_of_significant_digits:  5\n",
       "    units:                         m**2 s**-2\n",
       "    long_name:                     Geopotential\n",
-      "    standard_name:                 geopotential\n",
-      "Error: Skip state value for 'da_vars_attrs' since its content is not serializable\n"
+      "    standard_name:                 geopotential\n"
      ]
     },
     {
      "data": {
       "application/vnd.jupyter.widget-view+json": {
-       "model_id": "02bac65d50444882a40f189e2b8890e2",
+       "model_id": "7d9e374e0db240c5b08964da45a6a438",
        "version_major": 2,
        "version_minor": 0
       },
       "text/plain": [
-       "HTML(value='<iframe id=\"trame_trame__template_main\" src=\"http://localhost:43797/index.html?ui=main&reconnect=a…"
```

Comment:
if the trame-jupyter-extension were installed, that part will not be changing. Not a requirement, just an observation.

Reply:
Ah, this is leftover from the last time I ran the cell, and I forgot to clear the output. I must have bundled the change in with the automatic styling changes. I'll be rewriting the jupyter examples anyway.